Jenkinsfile

```diff
@@ -6,7 +6,8 @@ node {
   govuk.buildProject(
     sassLint: false,
     overrideTestTask: {
-      sh "bundle exec rake"
+      // Add Yarn bin directory to PATH so that phantomjs is available for the test suite
+      sh 'PATH="$(yarn bin):$PATH" bundle exec rake'
```

Comment:
This feels like a hacky way to change this environment variable.

I've seen `govuk.setEnvar` used elsewhere ([example](https://github.com/alphagov/manuals-publisher/blob/ba08a6746130f495cf430f8c5e246722f71d04f0/Jenkinsfile#L11)), but couldn't figure out how to get execute `yarn bin` from that context to prepend it to the variable.

If there's a more idiomatic way of achieving this, please let me know!

Reply:
I wonder if `overrideTestTask` is even needed anymore, since `buildProject` seems to run `bundle exec rake` anyway: https://github.com/alphagov/govuk-jenkinslib/blob/499503e3893ad184be923cddbd41fcc8b45d4897/vars/govuk.groovy#L664-L666

Reply:
@brucebolt I did wonder the same 🤔 but assumed it must have been there for a reason.

I can give it a go without – but only if there's a different way to set the `$PATH` environment variable.

Reply:
Maybe `yarn install` will set with `$PATH` env var?

Reply:
Unfortunately not. Yarn installs the packages into `node_modules/.bin` (same as NPM would), but it doesn't change the `$PATH` variable. Without that, the `phantomjs` command isn't available to the test runner.

Reply:
Looking into the need for `phantomjs`, it appears the only reason Rails doesn't install it automatically is because it was switched off in this line: https://github.com/alphagov/govuk_admin_template/commit/b938bff6f76bc23232b523e0d2a688d34b9d6e22#diff-b69384ae492854b5477484a13cf190590f4278263b9983b53531898329ace4eeR4

The PR referenced in that commit message has now been merged, so it's probably safe to remove `spec/javascripts/support/jasmine_helper.rb`.  That means we wouldn't need to do any `phantomjs` installation at all (including adding the `yarn.lock` etc).

Reply:
Thanks for this @brucebolt! 🎉 This is much nicer than having to install it with Yarn.